OneCommunityGlobal/HighestGoodNetworkApp
# Changhao - Unit test for textsearchbox

# Description
Unit test for textsearchbox

## How to test:
1. check into current branch
2. do `npm install` and `npm test TextSearchBox.test.js` to run this PR locally
3. check all 6 tests passed

## Screenshot:
<img width="752" alt="Screenshot 2023-12-07 at 11 00 29 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/516646c4-3667-498e-8e24-9326de2eced4">
